salt/cli/batch.py

```diff
@@ -312,6 +312,11 @@ def run(self):
                     if self.opts.get('failhard') and data['ret']['retcode'] > 0:
                         failhard = True
 
+                # avoid an exception if the minion does not respond.
+                if data.get("failed") is True:
+                    log.debug('Minion %s failed to respond: data=%s', minion, data)
+                    data = {'ret': 'Minion did not return. [Failed]', 'retcode': 1}
```

Comment:
I'm ok except the hardcoded retcode value. Could you please use a constant instead?
```
'retcode': salt.defaults.exitcodes.EX_GENERIC
```

Reply:
Indeed. You're right. I forgot to search for a constant.

Reply:
I've pushed a new commit fixing this